smart-contracts/justfile

```diff
@@ -34,4 +34,11 @@ workspace-optimize:
             --mount type=volume,source="$(basename "$(pwd)")_cache",target=/target \
             --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
             --platform linux/amd64 \
-            cosmwasm/workspace-optimizer:0.16.0; fi
+            cosmwasm/optimizer:0.16.0; fi
+
+optimize-single-contract:
```

Comment:
I would call that `contract-optimize` to maintain some consistency with the above command.

Reply:
removed, that did slip through from previous tests.
While that is convenient, we don't need it here